webpack/webpack.common.js

```diff
@@ -108,6 +110,8 @@ module.exports = async options => {
             SERVER_API_URL: `''`,
             DOCKER: options.DOCKER,
           },
+          'Date.now': process.env.DOCKER ? `() => ${DEFAULT_DATE}` : 'Date.now',
+          'new Date()': process.env.DOCKER ? `new Date(${DEFAULT_DATE})` : 'new Date()',
```

Comment:
Will this be picked up if we run tests locally and not through docker?

Reply:
This will not. I think it's just an edge case that we should leave alone (and not introduce additional complexity). The tests will fail anyway if run locally since the screen size will be off. The only reason I would ever run the tests locally is during development so you don't need to keep rebuilding the image. Running the tests locally should always be expected to fail.